template/template/chains.py

```diff
@@ -0,0 +1,8 @@
+""" Module to store Chain class """
+from enum import Enum
+
+
+class Chains(Enum):
+    MYCHAIN1 = 1
+    MYCHAIN2 = 1
```

Comment:
Didn't we move the chain to the `sdk_commons`?
The example should not use the same number `1` :)

Reply:
This is a very good question.
At the moment this Enum is directly included in the config to define the supported chains and both Ribbon and Opyn are using it to verify the input chain_id
```
        if config.chain_id not in Chains:
            raise ValueError("Invalid chain")
```

So having a common Chains would require to change some things

Please note that we also have a list of supported_chains our side, probably we want to normalize all of this, but I'm not totally sure about the best pattern to be used here. Wdyt?

Reply:
We want to have a single source of truth,
and have partners not proposing their own in case.

We also have control on what gets merged in the repo,
so we can definitely rely on the chain definition to be in here also for our internal code.

@cassios to also provide feedback.

Reply:
Well, I agree! But that would require some changes to the SDKs.
Since this PR is built on top of the existing code without touching the underlying implementation I prefer to introduce this change on a separated PR to prevent any confusion

Reply:
Agree on the separate PR.
A new task in https://tradeparadigm.atlassian.net/browse/PTP-9832 ?

Reply:
Sure, done